.github/workflows/nightly.yml

```
@@ -0,0 +1,25 @@
+name: Rust nightly CI
```

Aligning with other task names. This is evidently Rust.

```suggestion
name: Nightly CI
```